.gitignore

```diff
@@ -7,3 +7,4 @@ MANIFEST
 __pycache__
 /coverage/
 /.coverage
+.idea/
```

Comment:
this is not related

Reply:
Wouldn't you like to have the JetBrains project directory ignored>